packages/versioning/src/repository.ts

```diff
@@ -98,6 +99,26 @@ export class SchemaRepositoryVersion {
     });
   }
 
+  static async generateReadableSchema({
```

Comment:
maybe you could return the bundled | dereferenced schema ?
in case we want to chain this function with others.